### PR TITLE
add connection string for plan replayer

### PR DESCRIPTION
**Pull request by terry1purcell**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Resolve issue: #16497

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): 
- Issue #16947

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dveeden — APPROVED** [review]

File: sql-plan-replayer.md [hunk 1]
```diff
@@ -196,6 +196,11 @@ mysql> show stats_meta;
 
 After the scene is loaded and restored, you can diagnose and improve the execution plan for the cluster.
 
+> **Note:**
+>
+> If you use `mysql` and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```
Comment (mjonss) [inline sql-plan-replayer.md:201]:
```suggestion
> If you use `mysql` command line utility, and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```

**Review by mjonss — APPROVED** [review]
LGTM, with a minor suggestion.

File: sql-plan-replayer.md (hunk 1 above)
Comment (terry1purcell) [reply sql-plan-replayer.md:201]: This text was copied from the FAQ section of https://docs.pingcap.com/tidbcloud/tidb-cloud-import-local-files. Should I add "command line utility" there also? Thanks

Comment (mjonss) [reply sql-plan-replayer.md:201]: Why not :) does not hurt to be more clear. Grepping "command line" and "mysql" gave some examples like `MySQL command line` in: https://docs.pingcap.com/tidbcloud/sql-statement-kill#mysql-compatibility
But it is not strictly necessary, it is mostly a suggestion :)

Comment (terry1purcell) [reply sql-plan-replayer.md]: Totally agree! Thanks for the suggestion. And.....done.

File: sql-plan-replayer.md [hunk 2]
```diff
@@ -196,6 +196,11 @@ mysql> show stats_meta;
 
 After the scene is loaded and restored, you can diagnose and improve the execution plan for the cluster.
 
+> **Note:**
+>
+> If you use `mysql` command line utility and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
+
+
```
Comment (qiancai) [inline sql-plan-replayer.md:203]:
```suggestion
```

File: sql-plan-replayer.md [hunk 3]
```diff
@@ -196,6 +196,11 @@ mysql> show stats_meta;
 
 After the scene is loaded and restored, you can diagnose and improve the execution plan for the cluster.
 
+> **Note:**
+>
+> If you use `mysql` command line utility and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```
Comment (qiancai) [inline sql-plan-replayer.md:201]:
```suggestion
> If you use the `mysql` command-line client and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ CREATE TABLE `import_test` (
 LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
 ```
 
-If you use `mysql` and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
+If you use `mysql` command line utility and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-import-local-files.md:101]:
```suggestion
If you use the `mysql` command-line client and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**